### PR TITLE
Recent templates

### PR DESCRIPTION
**Pull request by eozan**
Tests and fix for broken tests coming soon.

### CODE REVIEW

**Comment by plamut** [comment]
Tests should be added, otherwise it seems fine.
(and extra points if docstrings are added ;) )

File: client/app/scripts/superdesk-templates/templates.js [hunk 1]
```diff
@@ -54,6 +55,57 @@
...
+
+            var criteria = [];
+            _.each(templateIds, function(id) {
+                criteria.push({_id: id});
+            });
+
+            var params = {
+                max_results: PAGE_SIZE,
+                page: 1,
+                where: JSON.stringify({'$or': criteria})
```
Comment (petrjasek) [inline client/app/scripts/superdesk-templates/templates.js]: fyi there is also `{_id: {$in: templateIds}}` query

**Comment by petrjasek** [comment]
looks fine, just wondering are those last used templates ordered somehow?

**Comment by eozan** [comment]
Last used template is added to the top of the list, That's the only ordering used here.

**Comment by petrjasek** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**